dashboard/config/scripts_json/csp1-2020.script_json

```diff
@@ -293,11 +293,11 @@
     },
     {
       "key": "Lesson 14: Assessment Day",
-      "name": "Lesson 14: Assessment Day",
+      "name": "Assessment Day",
       "absolute_position": 15,
       "lockable": true,
-      "has_lesson_plan": false,
-      "relative_position": 2,
+      "has_lesson_plan": true,
+      "relative_position": 14,
```

Comment:
It's fine that you included these, but just to make sure we're on the same page -- because these are not migrated scripts, it is the .script files (not the .script_json files) that will control what data gets seeded into production with this change.

Reply:
Yup the source of truth for these changes are the script files. I just included these updates because it felt good to try to keep the script_json as up to date as possible